### PR TITLE
Fix for 4149

### PR DESCRIPTION
**Pull request by ChamaraPhilipsuom**
Moved certificate alias drop down up in the spconfig.

### CODE REVIEW

File: components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp [hunk 1]
```diff
@@ -903,6 +896,70 @@ function clearAll() {
...
+    <td>
+        <fmt:message key="sp.certAlias"/>
+    </td>
+    <td>
+        <select id="alias" name="alias">
+            <%
+                if (aliasSet != null) {
+                    for (String alias : aliasSet) {
+                        if (alias != null) {
+                            if (alias.equals(provider.getCertAlias())) {
```
Comment (pulasthi7) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp]: the two if conditions can be merged to a single one

File: components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp [hunk 2]
```diff
@@ -903,6 +896,70 @@ function clearAll() {
...
+    <td>
+        <fmt:message key="sp.certAlias"/>
+    </td>
+    <td>
+        <select id="alias" name="alias" >
+            <%
+                if (aliasSet != null) {
+                    for (String alias : aliasSet) {
+                        if (alias != null) {
+                            if (alias.equals(samlSsoServuceProviderConfigBean.getCertificateAlias())) {
```
Comment (pulasthi7) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp]: the two if conditions can be merged to a single one

File: components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp (hunk 1 above)
Comment (ChamaraPhilipsuom) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp]: Fixed at #1682

File: components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp (hunk 2 above)
Comment (ChamaraPhilipsuom) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp]: Fixed at #1682

**Pull request merged**